src/app/modules/data-set/data-set-project/data-set-project.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="data-set-project-container mat-elevation-z8 center-horizontally container" *ngIf="!isProjectsEmpty()">
-    <mat-form-field appearance="standard">
+    <mat-form-field appearance="standard" class="search">
         <mat-label>Search by name</mat-label>
-        <input matInput (keyup)="searchProjects($event)" #input>
+        <input matInput (keyup)="searchProjects($event)" placeholder="My project">
```

Comment:
Can we ignore case in search? This is already implemented for the instances table.